src/app.jsx

```diff
@@ -545,11 +547,28 @@ class Application extends React.Component {
                     }
                 });
 
+        cockpit.spawn(["systemctl", "--user", "show", "--value", "-p", "LoadState", "podman-restart"], { environ: ["LC_ALL=C"], error: "ignore" })
+                .then(out => {
+                    if (out.trim() === "loaded") {
+                        this.setState({ userPodmanRestartAvailable: true });
+                    } else {
+                        this.setState({ userPodmanRestartAvailable: false });
```

Comment:
Unify to `userPodmanRestartAvailable: out.trim() === "loaded"`

Reply:
Looks good, thanks!